cavityDisplayBackend.py

```diff
@@ -1,12 +1,16 @@
+# from epics import PV
 from time import time
 
 from displayCavity import DISPLAY_LINAC_OBJECTS
 
+# WATCHER_PV = PV("PHYS:SYS0:1:SC_CAV_FAULT_HEARTBEAT")
+
 while True:
     start = time()
     for linac in DISPLAY_LINAC_OBJECTS:
         for cryomodule in linac.cryomodules.values():
             for cavity in cryomodule.cavities.values():
                 cavity.runThroughFaults()
 
+    # WATCHER_PV.put(WATCHER_PV.value + 1)
```

Comment:
Is this the pv that will be monitored externally? (I just tried to caget it and see that it does not yet exist, which explains why it's commented out!) Will the "watcher" be constantly monitoring to make sure that the WATCHER_PV value never stops increasing?

Reply:
Yup!